### ISSUE TITLE
Full sync from Genesis cosmovisor error 

### ISSUE THREAD

**Issue by alexanderkent** [zigbee-alliance/distributed-compliance-ledger#516]
Attempting to sync a full node from genesis using v0.12.0 on main-net, runs for a bit but then crashes with:

```
cosmovisor[18918]: 4:16PM INF running app args=["start"] module=cosmovisor path=/root/.dcl/cosmovisor/genesis/bin/dcld
cosmovisor[18918]: 4:16PM ERR failed to read error="lstat /root/.dcl/cosmovisor/current/upgrade-info.json: no such file or directory" filename=/root/.dcl/cosmovisor/current/upgrade-info.json module=cosmovisor
cosmovisor[18924]: 4:16PM INF starting ABCI with Tendermint
cosmovisor[18924]: 4:16PM INF Starting multiAppConn service impl=multiAppConn module=proxy
cosmovisor[18924]: 4:16PM INF Starting localClient service connection=query impl=localClient module=abci-client
cosmovisor[18924]: 4:16PM INF Starting localClient service connection=snapshot impl=localClient module=abci-client
cosmovisor[18924]: 4:16PM INF Starting localClient service connection=mempool impl=localClient module=abci-client
cosmovisor[18924]: 4:16PM INF Starting localClient service connection=consensus impl=localClient module=abci-client
cosmovisor[18924]: 4:16PM INF Starting EventBus service impl=EventBus module=events
cosmovisor[18924]: 4:16PM INF Starting PubSub service impl=PubSub module=pubsub
cosmovisor[18924]: 4:16PM INF Starting IndexerService service impl=IndexerService module=txindex
cosmovisor[18924]: 4:16PM INF ABCI Handshake App Info hash="�56���\x19@ ťh��\x1cRA�\x18�u8#�\x00����t�\v" height=46415 module=consensus protocol-version=0 software-version=0.12.0
cosmovisor[18924]: 4:16PM INF ABCI Replay Blocks appHeight=46415 module=consensus stateHeight=46415 storeHeight=46416
cosmovisor[18924]: 4:16PM INF Replay last block using real app module=consensus
cosmovisor[18924]: Error: error during handshake: error on replay: wrong Block.Header.AppHash.  Expected 863536F4B5AC194020C5A568E9CB1C5241B418E2753823E40088B8A7AF74FD0B, got 480D25B8475DA40AC0E318A5D3F0A64A0D4C0BA92E3F281C8BB37863375758AB
cosmovisor[18924]: Usage:
cosmovisor[18924]:   dcld start [flags]
```

Are there any recent breaking changes? What's the guidance/steps to sync from genesis?

**Comment by 6062055** [comment]
+1 
I have the same problem 😞

**Comment by alexanderkent** [comment]
Resolved with 0.12.1
